packages/common/container-definitions/src/runtime.ts

```diff
@@ -121,7 +121,6 @@ export interface IBatchMessage {
  * and the Container has created a new ContainerContext.
  */
 export interface IContainerContext extends IDisposable {
-    readonly existing: boolean | undefined;
```

Comment:
I do not think we can remove it without it being deprecated in previous release (and ensure that previous release is not using it)